northstar/src/runtime/minijail.rs

```diff
@@ -497,24 +497,26 @@ impl Debug {
     /// Start configured debug facilities and attach to `pid`
     async fn from(config: &Config, manifest: &Manifest, pid: u32) -> Result<Debug, Error> {
         // Attach a strace instance if configured in the runtime configuration
-        let strace = if let Some(strace) = config
+        let strace: OptionFuture<_> = config
             .debug
             .as_ref()
             .and_then(|debug| debug.strace.as_ref())
-        {
-            Some(Strace::new(strace, manifest, &config.log_dir, pid).await?)
-        } else {
-            None
-        };
+            .map(|strace| Strace::new(strace, manifest, &config.log_dir, pid))
+            .into();
```

Comment:
Didn't know about this OptionFuture...works but does not make the code more readable...or is the behavior changed in any way?

Reply:
Nope, no difference at all. The reason for this change is to address a complain from clippy nightly that we should use `Option::map` here :man_shrugging: